staging/provisioning/windows/kubeproxystart.ps1

```diff
@@ -24,7 +24,7 @@ if ($KubeproxyFeatureGates -contains "WinDSR=true") {
 }
 
 if ($KubeproxyFeatureGates.Count -ne 0) {
-    $global:KubeproxyArgList += @("--feature-gates='" + ($KubeproxyFeatureGates -join "','") + "'")
+    $global:KubeproxyArgList += @("--feature-gates=" + ($KubeproxyFeatureGates -join ","))
```

Comment:
nice catch, looks like those extra quote are not required from the docs: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#overview

Reply:
From my testing, even the `"` is not needed

Reply:
Ah, I should have caught this in the previous review :(
In powershell you can use backtick as an escape character if needed.

Reply:
Since these args are getting used to start a windows process single quotes will not work here (only powersell understands single quotes for windows). 
Double quotes or not quotes should both work but double quotes are safer in case any of the arguments contain spaces.

Reply:
From https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/, it should not contain spaces.